roles/matrix-bridge-mautrix-discord/templates/config.yaml.j2

```diff
@@ -101,7 +101,7 @@ bridge:
     message_error_notices: true
     # Should the bridge use space-restricted join rules instead of invite-only for guild rooms?
     # This can avoid unnecessary invite events in guild rooms when members are synced in.
-    restricted_rooms: true
+    restricted_rooms: {{ matrix_mautrix_discord_restricted_rooms }}
```

Comment:
This `bridge.restricted_rooms` setting should be controlled by a variabled named: `matrix_mautrix_discord_bridge_restricted_rooms`, not `matrix_mautrix_discord_restricted_rooms`.